### ISSUE TITLE
TypeError: search() got an unexpected keyword argument 'nb_result'

### ISSUE THREAD

**Issue by rcmpayne** [matlink/gplaycli#236]
- [x] Operating System (ie Ubuntu 16.04) 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.6 LTS
Release:        16.04
Codename:       xenial
- [x] Python version when running `gplaycli` (should be 3+) 
3.26
- [x] The way you installed `gplaycli`: `git clone`, `git clone and setup.py`, `pip install gplaycli`, `apt install gplaycli`, ...
Tried Both
- [x] The authentication method: (own) credentials or (own) token.
credentials

```
~/gplaycli$ gplaycli -v -c credentials.conf -s facebook
[INFO] GPlayCli version 3.26 [Python3.5.2]
[INFO] Configuration file is ['credentials.conf']
[INFO] Device is bacon
[INFO] Using credentials to connect to API
[INFO] Using plaintext password
Traceback (most recent call last):
  File "/home/user/.local/bin/gplaycli", line 11, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.5/site-packages/gplaycli/gplaycli.py", line 716, in main
    cli.search(args.search_string, nb_results, not args.paid)
  File "/home/user/.local/lib/python3.5/site-packages/gplaycli/hooks.py", line 9, in check_connection
    return function(self, *args, **kwargs)
  File "/home/user/.local/lib/python3.5/site-packages/gplaycli/gplaycli.py", line 340, in search
    results = self.api.search(search_string, nb_result=nb_results)
TypeError: search() got an unexpected keyword argument 'nb_result'
```

**Comment by matt668** [comment]
Check the version of gpaip you have installed (pip list)

Im guessing you have v0.4.4, in which case See #234 

and my branch that fixes it https://github.com/matt668/gplaycli/tree/Issue-234

**Comment by rcmpayne** [comment]
> Check the version of gpaip you have installed (pip list)
> 
> Im guessing you have v0.4.4, in which case See #234
> 
> and my branch that fixes it https://github.com/matt668/gplaycli/tree/Issue-234
```
$ pip list
Package      Version
------------ ----------
certifi      2019.11.28
cffi         1.13.2
chardet      3.0.4
Click        7.0
cryptography 2.8
idna         2.8
lxml         4.4.2
pip          19.3.1
protobuf     3.11.1
pyaxmlparser 0.3.24
pycparser    2.19
requests     2.22.0
setuptools   41.2.0
six          1.13.0
urllib3      1.25.7
```
```
$ pip install gpapi
Installing collected packages: gpapi
Successfully installed gpapi-0.4.4
$ python --version
Python 3.8.0

Using c:\users\rcmpa\appdata\local\programs\python\python38-32\lib\site-packages
Searching for pycparser==2.19
Best match: pycparser 2.19
Adding pycparser 2.19 to easy-install.pth file

rcmpa@ MINGW64 /c/Code/gplaycli ((82c5709...))
$ python setup.py install

Using c:\users\rcmpa\appdata\local\programs\python\python38-32\lib\site-packages
Finished processing dependencies for GPlayCli==3.26

$ pip list
Package      Version
------------ ----------
gpapi        0.4.4
gplaycli     3.26

```
```
$ gplaycli -c /c/Code/gplaycli/gplaycli.conf -v -s facebook
[INFO] GPlayCli version 3.26 [Python3.8.0]
[INFO] Configuration file is ['C:/Code/gplaycli/gplaycli.conf']
[INFO] Device is bacon
[ERROR] Cache file does not exists or is corrupted
[INFO] Retrieving token ...
[INFO] Token URL is https://matlink.fr/token/email/gsfid/bacon
[ERROR] Unknown error: <html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>

```

**Comment by matt668** [comment]
Try using classic credentials (set the email and password in the config file) Matlinks Token dispenser is currently offline

**Comment by rcmpayne** [comment]
I am. when i switch to your branch i start getting [INFO] Retrieving token .. even though i have the config set for username and password.

**Comment by matt668** [comment]
Yes I see,

There is a bug in the matlinks master, which is what my branch is forked off.

Line 602 of gplaycli.py should read

	`parser.add_argument('-t',  '--token', help="Instead of classical credentials, use the tokenize version", action='store_true', default=**False**)`

(change the true at the end to false)

Issue #233 identified it, although it was not fixed

**Comment by muzzyrost** [comment]
> Yes I see,
> 
> There is a bug in the matlinks master, which is what my branch is forked off.
> 
> Line 602 of gplaycli.py should read
> 
> `parser.add_argument('-t', '--token', help="Instead of classical credentials, use the tokenize version", action='store_true', default=False)`
> 
> (change the true at the end to false)
> 
> Issue #233 identified it, although it was not fixed

Thanks for fix!

> Check the version of gpaip you have installed (pip list)
> 
> Im guessing you have v0.4.4, in which case See #234
> 
> and my branch that fixes it https://github.com/matt668/gplaycli/tree/Issue-234

Thanks for fix, now it works.

**Comment by mirabilos** [comment]
This also happens with the gplaycli version in Debian unstable. Corresponding bug report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950114

**Comment by matlink** [comment]
Fixed in 3.27
